private/rules/maven_project_jar.bzl

```diff
@@ -60,6 +72,12 @@ def _maven_project_jar_impl(ctx):
         ctx.attr.excluded_workspaces,
     )
 
+    # Strip protobuf runtime
+    artifact_jars = _strip_toolchain_protobuf_runtime(
```

Comment:
If the `java_export` depends on protobuf, we really should add a dependency on `protobuf-java` rather than just stripping it. The question then becomes how to allow the user to set that.

Reply:
Neither do I. Let's leave it for now